src/platform/elements/form/Control.ts

```diff
@@ -231,323 +233,323 @@ export class NovoCustomControlContainerElement {
             </div>
         </div>
     `,
-  host: {
-    '[class]': 'form.controls[control.key].controlType',
-    '[attr.data-control-type]': 'form.controls[control.key].controlType',
-    '[class.disabled]': 'form.controls[control.key].readOnly',
-    '[class.hidden]': 'form.controls[control.key].hidden',
-    '[attr.data-control-key]': 'control.key',
-  },
+    host: {
+        '[class]': 'form.controls[control.key].controlType',
+        '[attr.data-control-type]': 'form.controls[control.key].controlType',
+        '[class.disabled]': 'form.controls[control.key].readOnly',
+        '[class.hidden]': 'form.controls[control.key].hidden',
+        '[attr.data-control-key]': 'control.key',
+    },
 })
 export class NovoControlElement extends OutsideClick implements OnInit, OnDestroy, AfterViewInit {
-  @Input() control: any;
```

Comment:
Spacing? Do you not use an editor-config plugin?

Reply:
it was because of the editor-config plugin. some files use 4 spaces and some use 2 - some files have mixed spacing. Isn't our standard 4 spaces? i can update editor-config to use 4 instead of 2.